ionos_sbx/bacula/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bacula
+  namespace: bacula
+  labels:
+    app.kubernetes.io/instance: bacula
+    app.kubernetes.io/name: bacula
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: bacula
+      app.kubernetes.io/name: bacula
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: bacula
+        app.kubernetes.io/name: bacula
+    spec:
+      containers:
+        - name: bacula
```

Comment:
General recommendation - I suggest to put the name of type of resource to the end,i.e:
bacula-deployment.yaml
bacula-ingress.yaml
bacula-dir-service.yaml
bacula-sd-service.yaml